index.js

```diff
@@ -44,7 +44,6 @@ class ClinicBubbleprof extends events.EventEmitter {
 
     if (this.detectPort) {
       logArgs.push('-r', 'detect-port.js')
-      stdio.push('pipe')
```

Comment:
why this change?

Reply:
@mcollina - We are creating the pipe in the array instead. 
`const stdio = ['inherit', 'inherit', 'inherit', 'pipe']`

Both Port and Source Warning will be using stdio[3].